core/templates/dev/head/pages/topics_and_skills_dashboard/topics_and_skills_dashboard.html

```diff
@@ -1,2 +1,227 @@
+            ng-class="{'oppia-dashboard-tabs-active': activeTab === 'topics'}">
+          <a class="oppia-dashboard-tabs-text"
+             ng-click="setActiveTab('topics')"
+             translate="I18N_DASHBOARD_TOPICS">
+          </a>
+        </li>
+        <li ng-if="skillSummaries.length > 0"
+            ng-class="{'oppia-dashboard-tabs-active': activeTab === 'skills'}">
+          <a class="oppia-dashboard-tabs-text"
+             ng-click="setActiveTab('skills')"
+             translate="I18N_DASHBOARD_SKILLS">
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <div class="oppia-card-view-wrap" ng-if="activeTab === 'topics'">
+      <md-card ng-repeat="topic in topicSummaries"
+               class="oppia-activity-summary-tile oppia-dashboard-card-view-item">
+        <a ng-href="/topic_editor/<[topic.id]>">
+          <div class="title-section">
+            <h2 class="activity-title">
+              <span><[(topic.name | truncate:40)]></span>
+            </h2>
+          </div>
+          <div class="content-section">
+              Uncategorized Skills: <[topic.uncategorized_skill_count]>
+          </div>
+          <div class="content-section">
+              Canonical Stories: <[topic.canonical_story_count]>
+          </div>
+          <div class="content-section">
+              Additional Stories: <[topic.additional_story_count]>
+          </div>
+        </a>
+      </md-card>
+    </div>
+
+    <div class="oppia-card-view-wrap" ng-if="activeTab === 'skills'">
+      <md-card ng-repeat="skill in skillSummaries"
+               class="oppia-activity-summary-tile oppia-dashboard-card-view-item">
+        <a ng-href="/skill_editor/<[skill.id]>">
+          <div class="title-section">
+            <h2 class="activity-title">
+              <span><[(skill.description | truncate:40)]></span>
+            </h2>
+          </div>
+          <div class="content-section" style="margin-top: 25%; font-size: 0.9em;">
+              Misconceptions: <[skill.misconception_count]>
+          </div>
+        </a>
+      </md-card>
+    </div>
+  </div>
+  <style>
+  .oppia-dashboard-tabs {
+    padding-top: 7%;
+    border-bottom: 1px solid #eee;
+    display: -webkit-flex;
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 0;
+    padding-left: 0;
+    text-align: center;
+  }
+  .oppia-dashboard-tabs li {
+    display: -webkit-flex;
+    display: flex;
+    margin-bottom: 0;
+    width: 200px;
+  }
+  .oppia-dashboard-title {
+    color: #01645c;
+    font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+    font-size: 3em;
+    margin-top: 0px;
+    margin-bottom: 25px;
+    padding-top: 78.5px;
+    text-align: center;
+  }
+  .oppia-card-view-wrap {
+    position: relative;
+  }
+
+  .oppia-dashboard-card-view-item {
+    height: 250px;
+    margin-left: 10px;
+    margin-right: 10px;
+    position: static;
+    width: 184px;
+  }
+
+  a:hover {
+    text-decoration: none !important;
```

Comment:
Do not use !Important, and do not use a rule that extends beyond the scope of this page (e.g. to "a" tags everywhere). Every CSS rule on this page should have a prefix that scopes it to just this page. Shared CSS should go in oppia.css.

@apb7 -- any guidance to add here? (Lint checks should be catching stuff like this.) Also /cc @kevinlee12

Reply:
(Also @apb7 @kevinlee12 please let's file issues for these CSS lint checks if we're not tracking them already.)

Reply:
Hi @seanlip! We do have [`declaration-no-important`](https://stylelint.io/user-guide/rules/declaration-no-important/) rule but it has not been enabled since it was not a part of the recommended rules.
Regarding "Every CSS rule on this page should have a prefix that scopes it to just this page.", we require a custom rule.
I am filing an issue to extend the CSS rules and listing these two rules.
Thanks!

Reply:
OK, thanks! Let's file one also for alphabetization (see https://github.com/oppia/oppia/pull/5094/files/3ec3a53166e10481f5215eaf08939f20a6cc7c2f..cdf5d3df4aa5d3f7f1e99953eaacc966a145985a#r194923524).

Also, we need one for indentation in HTML? https://github.com/oppia/oppia/pull/5094/files/3ec3a53166e10481f5215eaf08939f20a6cc7c2f..cdf5d3df4aa5d3f7f1e99953eaacc966a145985a#r194923479

Reply:
Shall I add them to the same issue as they are related to CSS rules only?

Reply:
The remove underline worked only when I set !important, referred to this SO answer.
https://stackoverflow.com/questions/7188768/how-to-remove-underline-from-a-name-on-hover

Reply:
@apb7 sure, go ahead. (As long as they're kept track of, I don't really mind where; that's up to you.)